scripts/mkcloud

```diff
@@ -540,24 +540,84 @@ function setupadmin()
     fi
 }
 
+function shutdown_node() {
+    virsh shutdown $1
+}
```

Comment:
btw. `virsh start $cloud-admin` is still un-functionized a few lines below
So when we support a cloud in a cloud we will have to touch this code anyway. So again, I don't really see the point of growing the number of functions just for the sake of having a different name.